app/components/guide-section/subsection/template.hbs

```diff
@@ -16,6 +16,14 @@
         {{t "component.guide-section.section"}}
       </a>
       {{subsectionTitle}}
+      <a
+        href="https://github.com/ember-learn/ember-octane-vs-classic-cheat-sheet/edit/master/translations/{{@sectionId}}/{{subsectionId}}/{{this.intl.locale}}.yaml"
+        target="_blank"
+        class="edit-translation-link"
+        rel="noopener noreferrer"
+      >
+        {{t "component.guide-section.edit-translation-text"}}
```

Comment:
Thanks for adding the translation! Didn't specify this in the issue description but hoped that this would be done.